packages/sdk/typescript/human-protocol-sdk/test/operator.test.ts

```diff
@@ -211,15 +212,27 @@ describe('OperatorUtils', () => {
       const gqlFetchSpy = vi.spyOn(gqlFetch, 'default').mockResolvedValueOnce({
         leaders: [mockLeaderSubgraph, mockLeaderSubgraph],
       });
-      const filter = { chainId: ChainId.LOCALHOST, role: 'role' };
+      const filter: ILeadersFilter = {
+        chainId: ChainId.LOCALHOST,
+        role: 'role',
+      };
+      const first =
```

Comment:
Are these three lines useless? There are not `first`, `skip` and `orderDirection` in the object above